hack/rpm-deps.sh

```diff
@@ -105,6 +105,28 @@ bazel run \
     $libvirtdevel_base \
     $libvirtdevel_extra
 
+bazel run \
+    --config=${ARCHITECTURE} \
+    //:bazeldnf -- rpmtree \
+    --public --nobest \
+    --name sandboxroot_x86_64 \
+    --repofile rpm/centos-repo.yaml \
+    -f centos-stream-release \
+    $centos_base \
+    $centos_extra \
+    gcc python36
+
+bazel run \
+    --config=${ARCHITECTURE} \
+    //:bazeldnf -- rpmtree \
+    --public --nobest \
+    --name sandboxroot_aarch64 --arch aarch64 \
+    --repofile rpm/centos-repo.yaml \
+    -f centos-stream-release \
```

Comment:
Please use `--basesystem` instead of the deprecated `-f` here as well as in the previous call.

Reply:
Done.